packages/mongo/mongo.d.ts

```diff
@@ -195,6 +195,17 @@ export namespace Mongo {
       selector?: Selector<T> | ObjectID | string,
       options?: O
     ): Promise<DispatchTransform<O['transform'], T, U> | undefined>;
+    /**
+     * Gets the number of documents matching the filter. For a fast count of the total documents in a collection see `estimatedDocumentCount`.
+     * @param selector The query for filtering the set of documents to count
+     * @param options All options are listed in [MongoDB documentation](https://mongodb.github.io/node-mongodb-native/4.11/interfaces/CountDocumentsOptions.html). Please note that not all of them are available on the client.
+     */
+    countDocuments(selector?: Selector<T> | ObjectID | string, options?: MongoNpmModule.CountDocumentsOptions): Promise<number>;
+    /**
+     * Gets an estimate of the count of documents in a collection using collection metadata. For an exact count of the documents in a collection see `countDocuments`.
+     * @param options All options are listed in [MongoDB documentation](https://mongodb.github.io/node-mongodb-native/4.11/interfaces/CountDocumentsOptions.html). Please note that not all of them are available on the client.
+     */
+    estimatedDocumentCount(options: MongoNpmModule.EstimatedDocumentCountOptions): Promise<number>;
```

Comment:
The `options` should be an optional parameter.
```suggestion
    estimatedDocumentCount(options?: MongoNpmModule.EstimatedDocumentCountOptions): Promise<number>;
```

Reply:
Ah missed that! Thanks!